sql/parser/rename.go

```diff
@@ -58,3 +58,22 @@ func (node *RenameTable) String() string {
 	_, _ = buf.WriteString(fmt.Sprintf("%s RENAME TO %s", node.Name, node.NewName))
 	return buf.String()
 }
+
+func (*RenameIndex) statement() {}
```

Comment:
The `statement()` methods should be declared in `stmt.go`. I see this wasn't done elsewhere in this file. Do you mind cleaning that up?